vmware_dvs/agent/firewalls/vcenter_firewall.py

```diff
@@ -92,24 +87,29 @@ def security_group_updated(self, action_type, sec_group_ids,
                                device_id=None):
         pass
 
-    def _get_dvs_for_port_id(self, port_id, p_key=None):
-        # Check if port is already known
+    def _update_dvs_port_map(self, ports):
```

Comment:
You have only one call from get_dvs_for_port_id with only one port as argument. No need to use list object ports except port.